### PR TITLE
button component

### PR DESCRIPTION
**Pull request by RobLMartin**

### CODE REVIEW

File: src/components/Common/Elements/Button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
```
Comment (codyarose) [inline src/components/Common/Elements/Button/index.tsx]:
```suggestion
import React, { ReactNode } from "react";
```

File: src/components/Common/Elements/Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import styled from "styled-components";
+
+interface Props {
+  children: any;
```
Comment (codyarose) [inline src/components/Common/Elements/Button/index.tsx]:
```suggestion
  children: ReactNode;
```

Comment (codyarose) [reply src/components/Common/Elements/Button/index.tsx]: Giving children the `ReactNode` type will make it a bit more type safe than `any`
[An example explanation](https://egghead.io/lessons/react-use-react-reactnode-for-the-children-prop-in-react-typescript-components-and-render-props)

**Review by codyarose — CHANGES_REQUESTED** [review]
Make children prop more type safe

**Review by codyarose — APPROVED** [review]

**Review by raymondjosephsotto — APPROVED** [review]

**Pull request merged**